src/types/test-failed/override-for-non-existing-function-in-contract-from-base-trait.tact

```diff
@@ -0,0 +1,7 @@
+trait BaseTrait {
+    fun bar() {}
+}
+
+trait Test {
```

Comment:
this should be `contract Test`, right? the file is called `override-for-non-existing-function-in-contract-from-base-trait.tact`

Reply:
file should actually be called `override-for-non-existing-function-in-trait-from-base-trait.tact` :)
fixed